jenkins/github_helpers.py

```diff
@@ -152,12 +152,29 @@ def create_pull_request(self, repository, title, body, base, head, user_reviewer
                     reviewers=user_reviewers,
                     team_reviewers=team_reviewers,
                 )
+                self.verify_reviewers_tagged(pull_request, user_reviewers, team_reviewers)
+
             return pull_request
         except:
             raise Exception(
-                "Could not create pull request"
+                "Either pull request was not created or some reviewers were not tagged on PR"
```

Comment:
Could we also print the original exception's message here?  Right now we're throwing away information about exactly what the problem was.